fr/development/routing.rst

```diff
@@ -830,7 +830,7 @@ utilisés lors de la connexion des routes. Par exemple, si votre action
 renommer vos actions::
 
     $routes->resources('Articles', [
-        'actions' => ['update' => 'put', 'create' => 'create']
+        'actions' => ['update' => 'put', 'add' => 'create']
```

Comment:
They could be.
The key is `create` and the value can be anything.
@markstory For the sake of the example, maybe the value should be different ?